vanilla-framework/vanilla-framework
# 1609: Optimise icons

## Done

- Optimised Vanilla icons (regular + social)
- Lowered CI filesize threshold to 175000
- Fixed social icon size (from 42px to 40px)

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Check that the light icons, dark icons and social icons still look the same

## Details

Fixes #1609, fixes #1569
